app/controllers/users.js

```diff
@@ -0,0 +1,31 @@
+const users = require('../models').users,
+  bcrypt = require('bcryptjs'),
+  helpers = require('../helpers'),
+  logger = require('../logger'),
+  errors = require('../errors');
+
+exports.signUp = (req, res, next) => {
+  const saltRounds = 5;
+  const user = {
+    firstName: req.body.firstName,
+    lastName: req.body.lastName,
+    email: req.body.email,
+    password: req.body.password
+  };
+
+  helpers
+    .validateEmailPassword(user.email, user.password)
+    .then(pwd => {
+      return bcrypt.hash(pwd, saltRounds).then(hash => {
+        user.password = hash;
```

Comment:
create new user here, copying the values and overwriting de password. You can use ...